test/index.js

```diff
@@ -11,18 +11,29 @@ describe('unique stream', function() {
     s.readable = true;
 
     var n = 10;
-    var next = after(n, function () {
+    var next = after(n * 2, function () {
       setImmediate(function () {
         s.emit('end');
       });
     });
 
-    for (var i = 0; i < n; i++) {
-      var o = {
-        type: type,
-        name: 'name ' + i,
-        number: i * 10
-      };
+    for (var k = 0; k < n * 2; k++) {
+      var i = Math.floor(k / 2);
+      var o;
+      if (k % 2 === 0) {
+        o = {
+          type: type,
+          name: 'name ' + i,
+          number: i * 10,
+        };
+      }
```

Comment:
https://github.com/eugeneware/unique-stream/blob/07d13f763a1994883b5ee9738888225dbed94462/CONTRIBUTING.md#rules

> Contributors should attempt to adhere to the prevailing code-style.

`} else {` style is already used in this line. https://github.com/eugeneware/unique-stream/blob/40677d1ddc9efef117b6aff9f0d913c38e74131f/index.js#L7